include/crpropa/Grid.h

```diff
@@ -262,6 +271,16 @@ class Grid: public Referenced {
 	  By default this it the trilinear interpolation. The user can change the
 	  routine with the setInterpolationType function.*/
 	T interpolate(const Vector3d &position) {
+	// check for volume
```

Comment:
the whole section between `T interpolate(const Vector3d &position) {` needs an indent.